q2_feature_classifier/tests/test_classifier.py

```diff
@@ -54,6 +55,91 @@ def test_fit_classifier(self):
             right += ref[taxon].startswith(classified[taxon])
         self.assertGreater(right/len(classified), 0.95)
 
+    def test_populate_class_weight(self):
+        # should populate the class weight of a pipeline
+        weights = Artifact.import_data(
+            'FeatureTable[RelativeFrequency]',
+            self.get_data_path('class_weight.biom'))
```

Comment:
Would it be possible to use a BIOMV100 (JSON-era) BIOM file here? That way we aren't committing new binary files. If not, no worries.